packages/did-ipfs-store/src/didStore.ts

```diff
@@ -54,9 +55,19 @@ export class DidStore implements IDidStore {
   }
 
   async get(cid: string): Promise<string> {
-    const { data: content } = await axios.get(`${this.url}/ipfs/${cid}`, {
-      headers: this.headers,
-    });
+    let content: any;
+    try {
+      ({ data: content } = await axios.get(`${this.url}/ipfs/${cid}`, {
+        headers: this.headers,
+      }));
+    } catch (e) {
+      const { response } = <AxiosError>e;
+      if (response?.status === 504) {
```

Comment:
what do you think of adding this comment to explain why this check/conversion?
```suggestion
      // 504 is the expected response code when IPFS gateway is unable to provide content within time limit
      // https://github.com/ipfs/specs/blob/main/http-gateways/PATH_GATEWAY.md#504-gateway-timeout
      // In other words, this is the expected response code if the traversal of the DHT fails to find the content
      if (response?.status === 504) {
```